src/Gui/DlgSettingsNavigation.ui

```diff
@@ -234,7 +234,7 @@ Turntable: the part will be rotated around the z-axis.</string>
         <item row="2" column="0">
          <widget class="QLabel" name="newDocViewLabel">
           <property name="text">
-           <string>New document camera orientation</string>
+           <string>New document camera orientation / Home view</string>
```

Comment:
What about "Default camera orientation" here?

Reply:
Works for me.  We can be more descriptive in the tooltip.  I'll update the PR.

What about scaling when the user selects the Home view?  Currently, I have it using view fit all because it gives a reasonable zoom level, corrects any potential clipping issues, and centers the model on screen.

Reply:
Logically, in the pref settings you also have a zoom factor and if we are strict, we should use that zoom level. However, as your model grows, I find it interesting that the home view "adapts".... I would leave it like you did for now.